website/project/model.py

```diff
@@ -3484,7 +3484,7 @@ def terminate_embargo(self, auth):
             action=NodeLog.EMBARGO_TERMINATED,
             params={
                 'project': self._id,
-                'node': self._id,
+                'node': self.registered_from_id,
```

Comment:
Because you're changing the format of the saved log information, this will require a migration as well.